text/0000-bee-rfcs-readme/0000-bee-rfcs-readme.md

```diff
@@ -0,0 +1,58 @@
++ Feature name: `BEE RFCs readme motivation change`
++ Start date: 2019-09-26
++ RFC PR: [iotaledger/bee-rfcs#0000](https://github.com/iotaledger/bee-rfcs/pull/0000)
++ Bee issue: [iotaledger/bee#0000](https://github.com/iotaledger/bee/issues/0000)
+
+# Summary
+
+This RFC proposes a change to the RFC process itself. Particularly the `Motivation` section of the RFC template to put more emphasis on the need to write clear requirements and how to write them.
+
+# Motivation
+
++ A more consistent and better means of capturing the motivations driving the changes proposed in RFCs across all projects.
++ Ensuring the changes proposed in RFCs are thought through from the user and business perspective.
++ Stakeholders should be involved in the RFC process as early as a new RFC is created. A well defined motivation ensures that ideas can be easily validated before the design is scrutinized.
+
+# Detailed design
+
+Replace the current motivation in the template:
+```
+Why are we doing this? What use cases does it support? What is the expected outcome?
+```
+
+With the following:
+```
+Why are we doing this? What use cases does it support? What is the expected outcome?
+
+1. Write a summary of the motivation.
+2. List all the specific use cases that your proposal is trying to address. Write them from the perspective of the person who will be using the software. We recommend using the "Job story" format, which provides context, motivation, and the desired outcome.
```

Comment:
Replace this with:

2. List all the specific use cases that your proposal is trying to address.
3. Where applicable, write from the perspective of the person who will be using the software, for example using the "Job story" format:

Reply:
Done